test/unit/registry-routes-components.js

```diff
@@ -104,6 +115,97 @@ describe('registry : routes : components', function(){
     });
   });
 
+  describe('when making valid info request for two components', function(){
+
+    before(function(done){
+      initialise(mockedComponents['async-error2-component']);
+      componentsRoute = new ComponentsRoute({}, mockedRepository);
+
+      makeInfoRequest({
+        components: [{
+          name: 'async-error2-component',
+          version: '1.X.X'
+        }, {
+          name: 'async-error2-component',
+          version: '1.0.0'
+        }]
+      }, done);
+    });
+
+    it('should return 200 status code', function(){
+      expect(code).to.be.equal(200);
+    });
+
+    it('should return a response containing both components', function(){
+      expect(response.length).to.be.equal(2);
+    });
+
+    var expectedResponse = [{
+        status: 200,
+        response: {
+          name: 'async-error2-component',
+          type: 'oc-component',
+          requestVersion: '1.X.X',
+          version: '1.0.0',
+        }
+      }, {
+        status: 200,
+        response: {
+          name: 'async-error2-component',
+          type: 'oc-component',
+          requestVersion: '1.0.0',
+          version: '1.0.0',
+        }
+      }];
+
+    it('should return a response containing components in the correct order', function(){
+      expect(response).to.be.eql(expectedResponse);
+    });
+
+    // it('should return a response with error code and description for the first component', function(){
+    //   expect(response[0].response.code).to.be.equal('GENERIC_ERROR');
+    //   expect(response[0].response.error).to.be.equal('Component execution error: thisDoesnotExist is not defined');
+    // });
+
+    // it('should not return a response with rendered html for the second component', function(){
+    //   expect(response[1].response.html).to.be.undefined;
+    // });
+
+    // it('should include 500 status code for first component', function(){
+    //   expect(response[0].status).to.equal(500);
+    // });
+
+    // it('should include 200 status code for second component', function(){
+    //   expect(response[1].status).to.equal(200);
+    // });
+
+    // it('should return name and request version for both components', function(){
+    //   expect(response[0].response.name).to.be.equal('async-error2-component');
+    //   expect(response[0].response.requestVersion).to.be.equal('1.X.X');
+    //   expect(response[1].response.name).to.be.equal('async-error2-component');
+    //   expect(response[1].response.requestVersion).to.be.equal('1.0.0');
+    // });
+
+    // it('should rerurn the actual version for the second component', function() {
+    //   expect(response[1].response.version).to.be.equal('1.0.0');
+    // });
```

Comment:
Can we get rid of those comments? I think now it is correctly resolving the versions which is what I would expect from info endpoints so those tests are not necessary anymore